### PR TITLE
Adding missing item mods.

### PR DESCRIPTION
**Pull request by helixhamin**

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -22284,13 +22299,15 @@ INSERT INTO `item_mods` VALUES (18517,10,4);
...
+INSERT INTO `item_mods` VALUES (18519,73,-15);    -- Oneiros Axe: "Store TP"-15
+INSERT INTO `item_mods` VALUES (18519,169,-12);   -- Reduces movement speed
 INSERT INTO `item_mods` VALUES (18520,165,4);
-INSERT INTO `item_mods` VALUES (18522,10,5); -- vit
-INSERT INTO `item_mods` VALUES (18522,16,15); -- ice element
-INSERT INTO `item_mods` VALUES (18522,25,5); -- acc
-INSERT INTO `item_mods` VALUES (18523,10,6); -- vit
-INSERT INTO `item_mods` VALUES (18523,16,20); -- ice element
-INSERT INTO `item_mods` VALUES (18523,25,6); -- acc
+INSERT INTO `item_mods` VALUES (18522,10,5);      -- vit
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: Could you add the missing item name here since yer adjusting the comments?

File: sql/item_mods.sql [hunk 2]
```diff
@@ -22331,11 +22351,15 @@ INSERT INTO `item_mods` VALUES (18561,23,5);      -- Yhatdhara: Attack +5
...
 INSERT INTO `item_mods` VALUES (18562,48,9);      -- Weapon Skill Accuracy +9
+INSERT INTO `item_mods` VALUES (18565,8,15);      -- Adflictio: STR+15
 INSERT INTO `item_mods` VALUES (18571,119,20);    -- Daurdabla (Level 99): Singing skill +20
 INSERT INTO `item_mods` VALUES (18571,120,20);    -- String instrument skill +20
 INSERT INTO `item_mods` VALUES (18571,453,2);     -- Grants two additional song effects
 INSERT INTO `item_mods` VALUES (18571,454,30);    -- Increases song effect duration +30%
-INSERT INTO `item_mods` VALUES (18572,452,4);
+INSERT INTO `item_mods` VALUES (18572,14,10);     -- Gjallarhorn: CHR+10
+INSERT INTO `item_mods` VALUES (18572,119,25);    -- Singing skill +25
+-- INSERT INTO `item_mods` VALUES (18572,???,25); -- Wind instrument skill +25
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: Mod ID 121 is wind skill

File: sql/item_mods.sql (hunk 1 above)
Comment (helixhamin) [inline sql/item_mods.sql]: Fixed the comments, including item and ffxiah style markup for each modifier.

File: sql/item_mods.sql (hunk 2 above)
Comment (helixhamin) [inline sql/item_mods.sql]: Okay, for some reason, was thinking that was aero magic. Changed.

**Pull request merged**